libs/ledger-live-common/src/families/bitcoin/js-getTransactionStatus.ts

```diff
@@ -117,6 +117,7 @@ const getTransactionStatus = async (
     totalSpent,
     txInputs,
     txOutputs,
+    family: t.family,
```

Comment:
WIP, should be fine but it is sometimes annoying because AccountRaw only has currencyId and not the family. It is only retrieved when converting to account (because then the currency is found by id, and family is in it)